volmdlr/stl.py

```diff
@@ -304,10 +304,11 @@ def get_normals(self):
         for triangle in self.triangles:
             normal = triangle.normal()
             for point in triangle.points:
-                if point in list(points_normals.keys()):
+                try:
                     points_normals[point].append(normal)
-                else:
+                except KeyError:
                     points_normals[point] = [normal]
+
```

Comment:
Is this: 
>                 try:
>                     points_normals[point].append(normal)
>                 except KeyError:
>                     points_normals[point] = [normal]
more efficient than this ?

>                 if point in points_normals:
>                     points_normals[point].append(normal)
>                 else:
>                     points_normals[point] = [normal]

Reply:
What do you think @AxelRinghausen ?

Reply:
Actually, I think this version is the best:

> points_normals[point] = points_normals.get(point, []) + [normal] 

ps. [] is the _default value_

Reply:
Thanks for your advice ! I will add it into the code !

Reply:
I let the try / except, it seems to be more efficient than point in dict
I based my assumption on calculation time only